README.markdown

```diff
@@ -347,3 +347,15 @@ If setting the `default` alias does not establish the node version in new shells
 [3]: https://travis-ci.org/creationix/nvm
 [Urchin]: https://github.com/scraperwiki/urchin
 [Fish]: http://fishshell.com
+
+## Uninstall
+
+```
+rm -rf $NVM_DIR ~/.npm ~/.bower && \
```

Comment:
Sure, that's fine.

However, `npm cache clear` is how you'd get back to a clean npm state, and that should only be recommended as part of npm's troubleshooting.